### DISCUSSION TITLE
Tailwind 3 custom classes not working

### DISCUSSION THREAD

**Discussion by skaidigitalnorge** [tailwindlabs/tailwindcss#7964] [Help] [answered]
I've tried to upgrade one of my projects to Tailwind 3, but it for some reason, does not work. At first, none of the classes were applied. Suddenly it seemed to work, but after investigating, only the default classes provided by tailwind work, not my custom classes. My tailwind.config.js looks like this:
module.exports = {
  theme: {
...
  content: [
    "./pages/**/*.{js,ts,jsx,tsx}",
    "./components/**/*.{js,ts,jsx,tsx}",
  ],
};

I am using nextjs. I have the following dev dependencies in package.json
  "devDependencies": {
    "autoprefixer": "^10.4.0",
    "next-sitemap": "^1.6.199",
    "postcss": "^8.4.5",
    "prettier": "^2.6.1",
    "tailwindcss": "^3.0.7"
  }

**Comment by skaidigitalnorge** [comment]
Anyone?

**Comment by RobinMalfait** [comment]
Hey! Thank you for your bug report!
Much appreciated! 🙏

Please create a minimal reproduction repo so that we can help you.

**Reply by skaidigitalnorge** [reply]
I've removed everything except the essentials: 
https://github.com/skaidigitalnorge/test-repo-tailwind3

**Reply by skaidigitalnorge** [reply]
Did you get to take a look at it?

**Comment by adamwathan** [comment]
You've deleted all of the default Tailwind classes by overriding them in your config:

https://github.com/skaidigitalnorge/test-repo-tailwind3/blob/main/tailwind.config.js#L7-L287

Use `theme.extend` if you want to add new values while keeping the built in ones:

https://tailwindcss.com/docs/theme#extending-the-default-theme

**Reply by skaidigitalnorge** [reply]
I only want to use custom values. That's how I've done it with all my projects. 

In my example, primary-400 is a custom class that I want to use. 

Hope that made sense! Love using tailwind

**Comment by skaidigitalnorge** [comment]
Any ideas?

**Comment by skaidigitalnorge** [comment, marked as answer]
Lol. I had added an extra fontSizes at the top for some reason. Removing it fixed it!
